src/app/public/src/modules/edit-button/edit-button.component.scss

```diff
@@ -0,0 +1,3 @@
+button {
+  float: right;
```

Comment:
This style would probably be better to put in the wrapper component (`.stache-wrapper ::ng-deep .stache-btn-edit {}`). Otherwise, wherever we put the edit button, it will always float right (if we wanted to put the button somewhere else for example).

Reply:
Hmm, good point.